completion/tcolorboxlibraryfitting.cwl

```diff
@@ -3,26 +3,26 @@
 
 \tcboxfit{box content%text}
 \tcboxfit[options%keyvals]{box content%text}
-\newtcboxfit{cmd}{options%keyvals}#d
-\newtcboxfit{cmd}[args]{options%keyvals}#d
-\newtcboxfit{cmd}[args][default]{options%keyvals}#d
-\newtcboxfit[init options]{cmd}{options%keyvals}#d
-\newtcboxfit[init options]{cmd}[args]{options%keyvals}#*d
-\newtcboxfit[init options]{cmd}[args][default]{options%keyvals}#*d
+\newtcboxfit{cmd}{options%keyvals}#d1
+\newtcboxfit{cmd}[args]{options%keyvals}#d1
+\newtcboxfit{cmd}[args][default]{options%keyvals}#d1
+\newtcboxfit[init options]{cmd}{options%keyvals}#d1
+\newtcboxfit[init options]{cmd}[args]{options%keyvals}#*d1
+\newtcboxfit[init options]{cmd}[args][default]{options%keyvals}#*d1
 \renewtcboxfit{cmd}{options%keyvals}

```

Comment:
If `\renew...` commands alter argument specification, then they need the `d1` modifier too.

Note the current completion support for redefined commands is not perfect, see #3925.